package-kuring/Sources/UIKit/BotUI/BotView.swift

```diff
@@ -0,0 +1,161 @@
+//
+// Copyright (c) 2024 쿠링
+// See the 'License.txt' file for licensing information.
+//
+
+import SwiftUI
+import ComposableArchitecture
+import ColorSet
+import Networks
+import BotFeatures
+
+public struct BotView: View {
+    @Bindable var store: StoreOf<BotFeature>
+    @FocusState private var isInputFocused: Bool
+    @State private var isPopoverVisible = false
+    @State private var isSendPopupVisible = false
+    
+    public var body: some View {
+        ZStack {
+            Color.Kuring.bg
+                .ignoresSafeArea()
+                .onTapGesture {
+                    isInputFocused = false
+                }
+            
+            VStack(alignment: .center) {
+                headerView
+                chatView
+                inputView
+                infoText
+            }
+            .padding(.bottom, 16)
+            
+            if isSendPopupVisible {
+                sendPopup
+                    .transition(.opacity)
+                    .zIndex(1)
+            }
+        }
+    }
+    
+    private var headerView: some View {
+        HStack {
+            backButton
+            Spacer()
+            titleText
+            Spacer()
+            infoButton
+        }
+        .padding(.horizontal, 18)
+    }
+    
+    private var backButton: some View {
+        Button {
+            // 뒤로 가기 버튼 동작 구현
+        } label: {
+            Image(systemName: "chevron.backward")
+                .padding()
+                .frame(width: 20, height: 11)
+                .foregroundStyle(Color.black)
+        }
+    }
+    
+    private var titleText: some View {
+        Text("쿠링봇")
+            .padding()
+            .font(.system(size: 18, weight: .semibold))
+    }
+    
+    private var infoButton: some View {
+        Button {
+            isPopoverVisible.toggle()
+        } label: {
+            Image("icon_info_circle", bundle: Bundle.bots)
+        }
+        .popover(isPresented: $isPopoverVisible, arrowEdge: .top) {
+            popoverContent
+        }
+    }
+    
+    private var popoverContent: some View {
+        VStack(spacing: 10) {
+            Text("• 쿠링봇은 2024년 6월 이후의 공지\n  사항 내용을 기준으로 답변할 수 있\n  어요.")
+            Text("• 테스트 기간인 관계로 한 달에 2회\n  까지만 질문 가능해요.")
+        }
+        .lineSpacing(5)
+        .font(.system(size: 15, weight: .medium))
+        .padding(20)
+        .presentationBackground(Color.Kuring.gray100)
+        .presentationCompactAdaptation(.popover)
+    }
+    
+    private var chatView: some View {
+        if !store.state.chatHistory.isEmpty {
+            AnyView(ChatView(store: self.store))
+        } else {
+            AnyView(ChatEmptyView())
+        }
+    }
```

Comment:
```suggestion
    @ViewBuilder
    private var chatView: some View {
        if !store.state.chatHistory.isEmpty {
            ChatView(store: self.store))
        } else {
            ChatEmptyView()
        }
    }
```
추측이지만 뷰빌더 어노테이션을 추가하면 AnyView로 감싸지 않아도 되지 않을까 싶습니다.

Reply:
오홍 될수도 있겠네요 한번 시도해보겠습니다 !